doctrine/doctrine-bundle/1.6/manifest.json

```diff
@@ -9,8 +9,39 @@
     "env": {
         "#1": "Format described at https://www.doctrine-project.org/projects/doctrine-dbal/en/latest/reference/configuration.html#connecting-using-a-url",
         "#2": "For an SQLite database, use: \"sqlite:///%kernel.project_dir%/var/data.db\"",
-        "#3": "For a PostgreSQL database, use: \"postgresql://db_user:db_password@127.0.0.1:5432/db_name?serverVersion=11&charset=utf8\"",
+        "#3": "For a MySQL database, use: \"mysql://db_user:db_password@127.0.0.1:3306/db_name?serverVersion=5.7\"",
         "#4": "IMPORTANT: You MUST configure your server version, either here or in config/packages/doctrine.yaml",
-        "DATABASE_URL": "mysql://db_user:db_password@127.0.0.1:3306/db_name?serverVersion=5.7"
+        "DATABASE_URL": "postgresql://db_user:db_password@127.0.0.1:5432/db_name?serverVersion=13&charset=utf8"
+    },
+    "dockerfile": [
+        "RUN apk add --no-cache --virtual .pgsql-deps postgresql-dev; \\",
+        "\tdocker-php-ext-install -j$(nproc) pdo_pgsql; \\",
+        "\tapk add --no-cache --virtual .pgsql-rundeps so:libpq.so.5; \\",
+        "\tapk del .pgsql-deps"
+    ],
+    "docker-compose": {
+        "docker-compose.yml": {
+            "services": [
+                "database:",
+                "  image: postgres:${POSTGRES_VERSION:-13}-alpine",
+                "  environment:",
+                "    POSTGRES_DB: ${POSTGRES_DB:-app}",
+                "    # You should definitely change the password in production",
+                "    POSTGRES_PASSWORD: ${POSTGRES_PASSWORD:-ChangeMe}",
+                "    POSTGRES_USER: ${POSTGRES_USER:-symfony}",
+                "  volumes:",
+                "    - db-data:/var/lib/postgresql/data:rw",
+                "    # You may use a bind-mounted host directory instead, so that it is harder to accidentally remove the volume and lose all your data!",
+                "    # - ./docker/db/data:/var/lib/postgresql/data:rw"
+            ],
+            "volumes": ["db-data:"]
+        },
+        "docker-compose.override.yml": {
+            "services": [
+                "db:",
```

Comment:
Shouldn't this service be named `database` too?

Reply:
correct :) Up for a PR @pyrech?

Reply:
Yep :slightly_smiling_face:

Reply:
See https://github.com/symfony/recipes/pull/843 :wink:

Reply:
You're right, can you do a PR?